ossindex/ossindex.go

```diff
@@ -59,8 +60,13 @@ func getOssIndexUrl() string {
 
 func openDb(dbDir string) (db *badger.DB, err error) {
 	opts := badger.DefaultOptions
-	opts.Dir = dbDir + "/" + dbValueDirName
-	opts.ValueDir = dbDir + "/" + dbValueDirName
+	if flag.Lookup("test.v") == nil {
```

Comment:
So this feels real icky to me....i get why its here but using the `test.v` flag doesn't feel like the right thing to trigger on. If i run the tests with `go test` and dont add `-v` this path is not going to get hit. 

Maybe adding a custom Environment variable would make sense and the IQ tests that need it can set that Environment variable before they run the tests that. 

More intrusive and less icky :) change would be to figure out a way to pass in the DB into OSSIndex that way the tests could mock it out or tweak the path or whatever instead of needing this if check.

Reply:
Yeah I didn't really dig this either, but I did like what it accomplished. I can try and remove the v?

I agree long term the best way is to inject the DB into ossindex, that's what I had hoped for to be honest.

Reply:
Yeah if we can remove the `v` that would be a little better since it wont rely on the test run needing to be in verbose mode.

Reply:
In: ca93f03